pages/Addons.md

```diff
@@ -32,6 +32,7 @@ Name | Author(s) | Description | Info | GitHub | Download
 SlimeXpansion | John000708 | <details><summary>Click to expand</summary>This Addon - which is inspired by the mod Industrialcraft - adds a lot of new electrical machines. There is a Food Synthesizer, UU Matter, a Wireless Charger and more!</details> | [BukkitDev](https://dev.bukkit.org/projects/slimexpansion_v1-0)</details> | [Github](https://github.com/John000708/slimexpansion) | [Builds](https://thebusybiscuit.github.io/builds/John000708/SlimeXpansion/master/)
 SlimeScrap | John000708 | <details><summary>Click to expand</summary>This addon allows you to break down machines and other items into their components - useful for recycling old items.</details> | [BukkitDev](https://dev.bukkit.org/projects/slimescrap) | N/A | [BukkitDev](https://dev.bukkit.org/projects/slimescrap/files)
 SlimeCalculator | John000708 | <details><summary>Click to expand</summary>This Addon gives you the ability to calculate the resources needed to craft Slimefun items.</details> | [BukkitDev](https://dev.bukkit.org/projects/slimecalculator) | N/A | [BukkitDev](https://dev.bukkit.org/projects/slimecalculator/files)
+SoundMuffler | J3fftw1 | <details><summary>Click to expand</summary>This Addon adds machines that have the abbility to muffle sound.</details> | N/A | [Github](https://github.com/J3fftw1/SoundMuffler) | N/A
```

Comment:
*adds a machine which has the ability to muffle sound.

Reply:
Also, you should link straight to the releases page rather than the main repo.

Reply:
It should link to the main repo and not the releases page...
It's listed as "SOURCE CODE" Walshy.

Reply:
Oh I thought he did that under the download part LUL

Well, the download part should link right to releases